patches/blockprotocol+0.0.7.patch

```diff
@@ -33,49 +33,160 @@ index 20a9bbd..a849fa3 100644
+-  operation: BlockProtocolAggregateOperationInput;
+-};
+-
  export type BlockProtocolLink = {
--  linkId: string;
+   linkId: string;
    sourceAccountId?: string | null;
-   sourceEntityId: string;
+@@ -266,9 +261,12 @@ export type BlockProtocolLink = {
    sourceEntityTypeId?: string | null;
-@@ -304,14 +309,16 @@ export type BlockProtocolCreateLinksFunction = {
+   sourceEntityVersionId?: string | null;
+   destinationAccountId?: string | null;
++  destinationEntityId: string;
++  destinationEntityTypeId?: string | null;
++  destinationEntityVersionId?: string | null;
+   index?: number | null;
+   path: string;
+-} & (SingleTargetLinkFields | AggregationTargetLinkFields);
++};
+ 
+ export type BlockProtocolLinkGroup = {
+   sourceAccountId?: string | null;
+@@ -279,14 +277,6 @@ export type BlockProtocolLinkGroup = {
+   links: BlockProtocolLink[];
+ };
+ 
+-export type BlockProtocolLinkedAggregation = {
+-  sourceAccountId?: string | null;
+-  sourceEntityId: string;
+-  sourceEntityVersionId?: string | null;
+-  sourceEntityTypeId?: string | null;
+-  path: string;
+-} & BlockProtocolAggregateEntitiesResult<BlockProtocolEntity>;
+-
+ export type BlockProtocolGetLinkAction = {
+   linkId: string;
+ };
+@@ -304,14 +294,7 @@ export type BlockProtocolCreateLinksFunction = {
    (actions: BlockProtocolCreateLinksAction[]): Promise<BlockProtocolLink[]>;
  };
  
-+export type BlockProtocolUpdateLinkedAggregationActionFragment = {
-+  // temporary identifiers for LinkedAggregations - to be replaced with a single id
-+  sourceAccountId?: string | null;
-+  sourceEntityId: string;
-+  path: string;
-+}
-+
- export type BlockProtocolUpdateLinksAction =
-   | { linkId: string; data: Pick<BlockProtocolLink, "index"> }
+-export type BlockProtocolUpdateLinksAction =
+-  | { linkId: string; data: Pick<BlockProtocolLink, "index"> }
 -  | ({
 -      // temporary identifiers for LinkedAggregations - to be replaced with a single id
 -      sourceAccountId?: string | null;
 -      sourceEntityId: string;
 -      path: string;
 -    } & { data: BlockProtocolAggregateOperationInput });
-+  | (BlockProtocolUpdateLinkedAggregationActionFragment & { data: BlockProtocolAggregateOperationInput });
++export type BlockProtocolUpdateLinksAction = { linkId: string; data: Pick<BlockProtocolLink, "index"> }
  
  export type BlockProtocolUpdateLinksFunction = {
    (actions: BlockProtocolUpdateLinksAction[]): Promise<BlockProtocolLink[]>;
-@@ -452,4 +459,4 @@ export type BlockProtocolProps = {
-   linkedAggregations?: BlockProtocolLinkedAggregation[];
+@@ -327,6 +310,73 @@ export type BlockProtocolDeleteLinksFunction = {
+   (actions: BlockProtocolDeleteLinksAction[]): Promise<boolean[]>;
+ };
+ 
++// ---------------------- LINKED AGGREGATIONS ------------------------- //
++
++export type SingleTargetLinkFields = {
++  destinationEntityId: string;
++  destinationEntityTypeId?: string | null;
++  destinationEntityVersionId?: string | null;
++};
```

Comment:
This should now be unused - there are a few references in the codebase to `BlockProtocolLink & SingleTargetLinkFields`, but this can now simply be `BlockProtocolLink`

Reply:
Good point, thank you for spotting 👍

I've removed it in https://github.com/hashintel/hash/pull/497/commits/7436d4d35cbb33c85b4e64139e63872b9f598357